lib/ts/frontToken.ts

```diff
@@ -13,41 +13,41 @@ export default class FrontToken {
 
     private constructor() {}
 
-    static async getFrontToken(): Promise<string | null> {
-        if (!(await AuthHttpRequest.recipeImpl.doesSessionExist(AuthHttpRequest.config))) {
-            return null;
-        }
+    private static async getFrontTokenFromStorage(): Promise<string | null> {
+        let frontTokenFromStorage = await AsyncStorage.getItem(FRONT_TOKEN_KEY);
+
+        if (frontTokenFromStorage !== null) {
+            let value = "; " + frontTokenFromStorage;
```

Comment:
We don't need to change this now (maybe add an issue about it, I'm not sure), but this seems very specific to cookies and looks like something that should be done by `AsyncStorage`

Reply:
Im not sure what you mean here

Reply:
We get the front token from the storage using a specific key, but then do things like `value.split("; " + FRONT_TOKEN_NAME + "=");` to the value itself. This makes sense for cookies, but I'm not sure why we are including `FRONT_TOKEN_NAME` in the stored value otherwise.

Reply:
We store the front token as a cookie string in storage to be able to check for expiry. It could be stored in a simpler format but this is another one of those cases where we just did it this way to keep it similar to the website SDK